pixels/utils/auth.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import secrets
+
+from asyncpg import Connection
+from fastapi import HTTPException, Request
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from jose import JWTError, jwt
+
+from pixels.constants import Server
+from pixels.models import AuthState
+
+log = logging.getLogger(__name__)
+
+
+class JWTBearer(HTTPBearer):
+    """Dependancy for routes to enforce JWT auth."""
+
+    def __init__(self, auto_error: bool = True, is_mod_endpoint: bool = False):
+        super(JWTBearer, self).__init__(auto_error=auto_error)
```

Comment:
Can we not just do? What's the difference?
```suggestion
        super().__init__(auto_error=auto_error)
```